app/views/agents/rdv_wizard_steps/step2.html.slim

```diff
@@ -1,7 +1,8 @@
 - content_for :title do
   | Choisir le ou les usagers
 
-.card
+.card.js-rdv-step-card
+  .spinner.spinner-border
```

Comment:
I don't think you used the spinner so you can remove it altogether and the associated CSS